src/git.d.ts

```diff
@@ -0,0 +1,327 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
```

Comment:
Shouldn't we update this header? The license.txt will not be in the project root.